lib/charms/layer/apache_bigtop_base.py

```diff
@@ -19,12 +19,22 @@ class Bigtop(object):
     _roles = set()
     _overrides = {}
 
+    @property
+    def bigtop_base(self):
+        '''Path to the bigtop root directory.'''
+        return self._bigtop_base
+
+    @property
+    def site_yaml(self):
+        '''Path to the site_yaml config file.'''
+        return self._site_yaml
+
```

Comment:
(The self.site_yaml stuff will actually conflict when https://github.com/juju-solutions/layer-apache-bigtop-base/pull/24 gets merged -- will update this when that happens.)